Workbooks/CosmosDb/Cost Optimization/Optimizing Total Provisioned Throughput.workbook

```diff
@@ -0,0 +1,1002 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 1,
+            "content": {
+              "json": "<span style=\"font-size:40px;\"> Optimizing Total Provisioned Throughput </span>\r\n\r\n<br>\r\n\r\n<span style=\"font-size:30px;\"> Introduction </span>\r\n\r\n<br>\r\n<span style = \"font-size:15px;\">\r\nIf you’re currently facing high costs on Azure Cosmos DB, this workbook will walk you through troubleshooting why that might be the case. One of the most common reasons users experience higher costs than anticipated on Azure Cosmos DB is due to overprovisioning throughput.\r\n\r\n<span style = \"font-size:15px;\"> This workbook will help you:\r\n<br>\r\n<span style = \"font-size:15px;\">1.\tUnderstand what factors (at a high level) impact costs in Cosmos DB\r\n<br>\r\n<span style = \"font-size:15px;\">2.\tDetermine how much of your provisioned throughput you are utilizing \r\n<br>\r\n<span style = \"font-size:15px;\">3.\tMake the necessary changes to prevent overprovisioning and switch to manual or auto-scale based on current usage  \r\n"
```

Comment:
```suggestion
              "json": "<span style=\"font-size:40px;\"> Optimizing Total Provisioned Throughput </span>\r\n\r\n<br>\r\n\r\n<span style=\"font-size:30px;\"> Introduction </span>\r\n\r\n<br>\r\n<span style = \"font-size:15px;\">\r\n One common reason users experience higher costs than anticipated on Azure Cosmos DB is due to overprovisioning throughput. This workbook will help you troubleshoot why you may be experiencing higher costs with Azure Cosmos DB. \r\n\r\n<span style = \"font-size:15px;\"> In this workbook, we'll cover:\r\n<br>\r\n<span style = \"font-size:15px;\">1.\tUnderstanding what factors impact costs in Cosmos DB\r\n<br>\r\n<span style = \"font-size:15px;\">2.\tDetermining your throughput utilization \r\n<br>\r\n<span style = \"font-size:15px;\">3.\tHow to make the necessary changes to prevent overprovisioning and switching to manual or auto-scale based on your current usage  \r\n"
```